docker/python/pcloud.egg-info/PKG-INFO

```diff
@@ -0,0 +1,10 @@
+Metadata-Version: 1.0
```

Comment:
These files are not needed?

- PKG-INFO
- SOURCES.txt
- dependency_links.txt
- top_level.txt